scripts/pylint_extensions_test.py

```diff
@@ -49,10 +50,10 @@ def test_finds_non_explicit_keyword_args(self):
         checker_test_object.setup_method()
         func_call_node_one, func_call_node_two, func_call_node_three = (
             astroid.extract_node("""
-        def test(test_var_one, test_var_two=4, test_var_three=5, test_var_four="test_checker"):
+        def test(test_var_one, test_var_two=4, test_var_three=5,
+            test_var_four="test_checker"):
```

Comment:
Ditto.

Reply:
This was because pylint was throwing a warning, namely `line-too-long` (80 character limit).

Reply:
This should not be the case since initially, the former was passing all checks. Can you please look into this?

Reply:
ptal?